packages/blade/src/components/Tabs/TabItem.web.tsx

```diff
@@ -149,14 +149,17 @@ const TabItem = ({
             controls: panelId,
           })}
         >
-          {leading
-            ? React.cloneElement(leading as React.ReactElement, {
-                size: iconSizeMap[size!],
-                color: isSelected
+          {Leading ? (
+            <Leading
+              size={iconSizeMap[size!]}
+              // @ts-expect-error TODO: icons types don't support brand colors
```

Comment:
What is the todo action here? to add brand colors on icon?

or should this be internal usecase?

Reply:
We can also expose the brand colors, will discuss this with designers once.

Reply:
Cool. Maybe just create a github issue for now to keep track, we can discuss later. Not a blocker.

Reply:
Oh wait I think we already added this https://github.com/razorpay/blade/pull/1678